.github/workflows/pull_request.yaml

```diff
@@ -28,6 +28,7 @@ jobs:
           prettier: true
           eslint_extensions: ts
 
+
```

Comment:
Unnecessary empty line

Reply:
Due to a change in the dockerFile, it is important to check also if the new image is broken. Test this new image and see if this repo still works properly